src/index.tsx

```diff
@@ -8,7 +8,7 @@ import 'tapis-ui/index.css';
 
 ReactDOM.render(
   <React.StrictMode>
-    <TapisProvider basePath="https://tacc.tapis.io">
+    <TapisProvider basePath="https://dev.develop.tapis.io">
```

Comment:
This shouldn't change when going into production. We should work on getting your test projects and other data into the the tacc tenant.